Fix time date type issue, #12476 (#12477)
archesproject/arches: releases/8.0.6.md

Arches 8.0.6 Release Notes
--------------------------

### Bug Fixes and Enhancements

> - Fix date datatype bug that the datetime losese time components when the value is imported [#12476](https://github.com/archesproject/arches/pull/12476)

Reviewer: ```suggestion
- Fix date datatype bug in which a datetime value loses time details when it's imported [#12476](https://github.com/archesproject/arches/pull/12476)
```
